core/template.go

```diff
@@ -17,6 +22,12 @@ var TemplateFuncs = map[string]interface{}{
 		}
 		return ansi.ColorCode(color)
 	},
+	"HelpInputRune": func() string {
```

Comment:
I have addd these helper functions to the templates to allow for dynamic prompt configuration.  The way I had originally envisioned like:
```go
var PromptTemplate := `...` + HelpIcon + `...`
```
will not work because `PromptTemplate` is not redefined when user changes `HelpIcon`.  In order to get the template make `HelpIcon` and `HelpInputRune` dynamic I just added simple template wrappers to return the dynamic value.